map/bookmark.hpp

```diff
@@ -128,13 +122,13 @@ class BookmarkCategory : public UserMarkContainer
 
   void ClearTracks();
 
-  /// @name Tracks routine.
-  //@{
   void AddTrack(std::unique_ptr<Track> && track);
   Track const * GetTrack(size_t index) const;
   inline size_t GetTracksCount() const { return m_tracks.size(); }
   void DeleteTrack(size_t index);
-  //@}
+
+  std::vector<std::unique_ptr<Track>> && StealTracks() { return std::move(m_tracks); }
```

Comment:
Note that move from a vector leaves vector in unspecified-but-a-valid-state, so, to be sure, could you please clear m_tracks?